app/helpers/court_dates_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Helper methods for court_dates
+module CourtDatesHelper
+  def when_do_we_have_court_dates(casa_case)
+    date_query = casa_case.court_dates.includes(:hearing_type).ordered_ascending
```

Comment:
**nitpick:** why `date_query` over `court_dates`?

**question:** do we really need to include `hearing_type`?

Reply:
> why date_query over `court_dates`?

Because it was querying for the date and I was a bit too literal, I agree `court_dates` is much better

> do we really need to include `hearing_type`?

I don't think so. Removed it 🙂